posix.mak

```diff
@@ -750,21 +749,17 @@ $G/docs-latest.json : ${DMD_LATEST} ${DMD_LATEST_DIR} \
 # remove this after https://github.com/dlang/dmd/pull/7513 has been merged
 DMD_EXCLUDE_PRERELEASE =
 ifneq (,$(wildcard $(DMD_DIR)/src/dmd/objc_glue_stubs.d))
-   DMD_EXCLUDE_PRERELEASE += -e /objc_glue.d/d
+ DMD_EXCLUDE_PRERELEASE += -e /objc_glue.d/d
 endif
 
-# DDox tries to generate the docs for all `.d` files. However for dmd this is tricky,
-# because the `{mach, elf, mscoff}` are platform dependent.
-# Thus the need to exclude these files.
 $G/docs-prerelease.json : ${DMD} ${DMD_DIR} ${DRUNTIME_DIR} \
 		${PHOBOS_FILES_GENERATED} | dpl-docs
 	find ${DMD_DIR}/src -name '*.d' | \
-		sed -e /mscoff/d ${DMD_EXCLUDE_PRERELEASE} ${DMD_EXCLUDE} \
-			> $G/.prerelease-files.txt
-	find ${DRUNTIME_DIR}/src -name '*.d' | sed -e '/gcstub/d' \
-		-e /unittest/d >> $G/.prerelease-files.txt
-	find ${PHOBOS_DIR_GENERATED} -name '*.d' | sed -e /unittest.d/d \
-		-e /windows/d | sort >> $G/.prerelease-files.txt
+		sed ${DMD_EXCLUDE_PRERELEASE} > $G/.prerelease-files.txt
+	find ${DRUNTIME_DIR}/src -name '*.d' | \
+		sed -e /unittest/d >> $G/.prerelease-files.txt
+	find ${PHOBOS_DIR_GENERATED} -name '*.d' | \
+		sed -e /unittest.d/d | sort >> $G/.prerelease-files.txt
```

Comment:
I don't know why we need `sort` only for the files from Phobos, but I will try removing it in a separate PR.